schemas/iso19139/src/main/plugin/iso19139/process/thumbnail-remove.xsl

```diff
@@ -25,22 +25,36 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                 xmlns:gmd="http://www.isotc211.org/2005/gmd"
                 xmlns:gco="http://www.isotc211.org/2005/gco"
+                xmlns:xs="http://www.w3.org/2001/XMLSchema"
                 xmlns:geonet="http://www.fao.org/geonetwork" exclude-result-prefixes="#all"
+                xmlns:digestUtils="java:org.apache.commons.codec.digest.DigestUtils"
+                xmlns:exslt="http://exslt.org/common"
```

Comment:
```suggestion
                xmlns:exslt="http://exslt.org/common"
                exclude-result-prefixes="#all"
```

Otherwise the namespaces end in the `gmd:distributionInfo` element:

```
<gmd:distributionInfo xmlns:xs="http://www.w3.org/2001/XMLSchema"
                         xmlns:digestUtils="java:org.apache.commons.codec.digest.DigestUtils"
                         xmlns:exslt="http://exslt.org/common">
...
```

Reply:
Test is satisfied.

I made two same url+name resource. Only description is different.
![image](https://github.com/geonetwork/core-geonetwork/assets/74916635/0f9c2a66-2c36-41d4-af68-06123de58ad6)


Delete the google123 and the google_567 remains
![image](https://github.com/geonetwork/core-geonetwork/assets/74916635/54f1b2f7-0fdc-4149-b1ba-7d472bb52eb5)